lib/config/migrations/custom/trust-level-migration.ts

```diff
@@ -1,6 +1,7 @@
 import { AbstractMigration } from '../base/abstract-migration';
+import type { DeprecatedRenovateConfig } from '../types';
 
-export class TrustLevelMigration extends AbstractMigration {
+export class TrustLevelMigration extends AbstractMigration<DeprecatedRenovateConfig> {
```

Comment:
I don't think this change is necessary. You made `DeprecatedRenovateConfig` the default.

Reply:
Need to think how to optimize futher. Not reallly happy with that yet.